src/System.Management.Automation/engine/NativeCommandProcessor.cs

```diff
@@ -228,7 +228,7 @@ public override void GetObjectData(SerializationInfo info, StreamingContext cont
     /// <summary>
     /// Provides way to create and execute native commands.
     /// </summary>
-    internal class NativeCommandProcessor : CommandProcessorBase
+    internal partial class NativeCommandProcessor : CommandProcessorBase
```

Comment:
Why?

Reply:
LibraryImport requires it (otherwise you'll get compiler error)

Reply:
It seems the clas doesn't contain pinvokes.

Reply:
Good catch.  I had changed it when I tried to change `FindExecutable` to use LibraryImport, but since it doesn't work with `StringBuilder` and I couldn't get it to work as a buffer, I forgot to revert that as well.  Fixed.